tests/generation/test_generate_missing.py

```diff
@@ -39,13 +39,13 @@ def test_missing_steps():
 
     result.stdout.fnmatch_lines(
         [
-            'Step "I have a custom bar" is not defined in the scenario '
+            'Step "Given I have a custom bar" is not defined in the scenario '
```

Comment:
This step is defined in the python code as "I have a custom bar". It can also be used as "And/But I have a custom bar". But it will print out "Given I have.."... Isn't it better to format it differently to separate the actual name with quotes?
'Step Given "I have a custom bar" is not found' or something like this?